packages/utilities/psammead-assets/src/svgs.stories.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from 'styled-components';
+import { node, number, shape } from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { number as numberKnob, withKnobs } from '@storybook/addon-knobs';
+import notes from '../README.md';
+import { news, igbo, yoruba, thai, pidgin } from './svgs';
+
+const Svg = styled.svg`
+  display: block;
+  margin-top: 1.25rem;
+  fill: #fff;
+`;
+
+const Container = styled.div`
+  background: ${C_POSTBOX};
+  padding: 1rem;
+  height: 100vh;
+`;
+
+const getSVG = ({ group, ratio, viewbox, height }) => {
+  const width = height * ratio;
+
+  return (
+    <Container>
+      <Svg
+        xmlns="http://www.w3.org/2000/svg"
+        focusable="false"
+        aria-hidden="true"
+        viewBox={`0 0 ${viewbox.width} ${viewbox.height}`}
+        height={height}
+        width={width}
+      >
+        {group}
+      </Svg>
+    </Container>
+  );
+};
+
+getSVG.defaultProps = {
+  group: news.group,
+  ratio: news.ratio,
+  viewbox: {
+    width: 167.95,
+    height: 24,
+  },
+  height: 24,
+};
+
+getSVG.propTypes = {
+  group: node,
+  ratio: number,
+  viewbox: shape({
+    height: number,
+    width: number,
+  }),
+  height: number,
+};
+
+storiesOf('SVGs', module)
```

Comment:
Flicking between these on storybook, they seem to wiggle around a bit, were all of these not exported the same? I would expect them to all have the same height

Reply:
The svg's don't have the same height. I used the svgs as provided by @jblaut

Reply:
They seem to wiggle because they don't have the same aspect ratios.

Reply:
I would have thought the `BBC NEWS` in the svg would be a constant size/height/width and the only thing that would change would be the width of the whole svg

Reply:
we may need to ask for them exported the same height

Reply:
We can follow up on that